list-of-macros.md

```diff
@@ -223,6 +224,42 @@ We simply remove the circuit in environment 'circuitikz'.
 circuitikz
 
 
+## Package cleveref
+
+Source: [yalafi/packages/cleveref.py](yalafi/packages/cleveref.py),
+tests: [tests/test\_packages/test\_cleveref.py](tests/test_packages/test_cleveref.py)
+
+For YaLafi to support the `cleveref` package you must load it with the `poorman` option in your LaTeX document.
```

Comment:
Could you please break all lines to at most 79 characters? (For links, this seems to be impossible, though.)